documentation/src/test/java/example/registration/WebServerDemo.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+package example.registration
+
+import org.junit.jupiter.api.Assertions.assertEquals
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.api.extension.RegisterExtension
+
+// tag::user_guide[]
+class WebServerDemo {
+    // end::user_guide[]
+    // @formatter:off
+    // tag::user_guide[]
+    companion object {
+        @JvmField
+        @RegisterExtension
+        val server = WebServerExtension.builder()
+                .enableSecurity(false)
+                .build()
+    }
+    // end::user_guide[]
+    // @formatter:on
+    // tag::user_guide[]
+
+    @Test
+    fun getProductList() {
+        // end::user_guide[]
+        // tag::user_guide[]
```

Comment:
The above two lines can be removed, right?

Reply:
Yes. Good catch!

I just fixed that in the "origin": 35732161b6fc20d5825c1c049345f0e0e2eacd9e

Reply:
I'll change that here too.

Reply:
Thanks!